### PR TITLE
[Tools]Throw exception if a solution reference a not exist csproj.

### PR DESCRIPTION
**Pull request by wyunchi-ms**
### Problem want to solve
1. CI still pass when a referenced csproj file is removed. But this should be detected.

### Solution
1. We will detect the existance of every csproj file referenced by the solution files and throw exception if any of them doesn't exist.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/CreateFilterMappings.ps1 [hunk 1]
```diff
@@ -148,7 +148,7 @@ function Add-ProjectDependencies
         [string]$SolutionPath
     )
 
-    $CommonProjectsToIgnore = @("ScenarioTest.ResourceManager", "TestFx", "Tests" )
+    $CommonProjectsToIgnore = @("ScenarioTest.ResourceManager", "TestFx", "Tests", "Test", "src", "tools" )
```
Comment (msJinLei) [inline tools/CreateFilterMappings.ps1:151]: Can you use a more explicit condition to filter csproj wanted?

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Review by msJinLei — DISMISSED** [review]

**Review by BethanyZhou — APPROVED** [review]
LGTM

**Pull request merged**